SLACK-GUIDELINES.md

```diff
@@ -128,6 +128,15 @@ in the traditional sense.
 In the case that certain channels have rules or guidelines, they will be listed
 in the purpose or pinned docs of that channel.
 
+## Threaded Conversations
+We believe Slack threads in public channels are extremely useful for quickly and openly collaborating with your fellow contributors.  
+
+As a community, we value taking the time to assess and socialize important technical and community decisions. Ideally, concensus that is reached within Slack threads will make its way to less ephemeral and more comprehensive artifacts for sharing ideas and proposals. 
```

Comment:
I think "consensus" is misspelled here, but also I wonder if what we really mean here anyway is "context" (I don't think we'd ever take a slack thread as enough for consensus iyswim).

I'd actually be tempted to change the second sentence to something like "However, having conversations in slack can result in context being lost, because not everyone will read every thread and search and indexing isn't nearly as good as alternatives like GitHub issues, PRs and discussions. For this reason it's important that when slack discussions turn in to things that the broader community would benefit from or need to refer to or be aware of they're written up in a less ephemeral location (ie GitHub)." (Except probably less wordy; sorry, I'm terrible at being concise; anyway it's a thought :)).

Reply:
He, I had somethin similar typed up and forgot to send the `Comment` button :joy:. Same comment regarding "consensus" :).

Reply:
OK I am on it! I like this change, I'll wordsmith